### DISCUSSION TITLE
How to bulk update team entity

### DISCUSSION THREAD

**Discussion by stonekw** [Innofactor/XrmToolBox.BulkDataUpdater#223] [General]
I am trying to use the tool to bulk update a field on the Team entity, but when I attempt to do so, I get an error that the  'UpdateMultiple' method does not support entities of type 'team'. I don't see an option to update one record at a time in the execute dialog. Any help appreciated!

![image](https://github.com/rappen/BulkDataUpdater/assets/16467359/c3d1354a-23cd-40f8-83c1-d61b2a3a8e23)

![image](https://github.com/rappen/BulkDataUpdater/assets/16467359/7801123a-5a53-4416-9780-fda2fb012c54)

**Comment by rappen** [comment]
Thanks for the report.

When the "Batch size" is 1 (as in the screenshot above), it shouldn't try to use the `UpdateMultiple`...

I can reproduce the error. I will look at it.

**Reply by rappen** [reply]
Found the issue, solved it.
Until next release, you can uncheck the Set ISN, then it will work ok.
![image](https://github.com/rappen/BulkDataUpdater/assets/2572253/30030f5d-a258-496e-94f5-2940860b3e08)

**Reply by stonekw** [reply]
Thank you! Much appreciated. That works
